src/PlaywrightSharp/LifecycleWatcher.cs

```diff
@@ -12,9 +12,9 @@ internal class LifecycleWatcher : IDisposable
             new Dictionary<WaitUntilNavigation, string>
             {
                 [WaitUntilNavigation.Load] = "load",
-                [WaitUntilNavigation.DOMContentLoaded] = "DOMContentLoaded",
-                [WaitUntilNavigation.Networkidle0] = "networkIdle",
-                [WaitUntilNavigation.Networkidle2] = "networkAlmostIdle",
+                [WaitUntilNavigation.DOMContentLoaded] = "domcontentloaded",
+                [WaitUntilNavigation.Networkidle0] = "networkidle0",
+                [WaitUntilNavigation.Networkidle2] = "networkidle2",

```

Comment:
Why are we changing this?

Reply:
this - https://github.com/kblok/playwright/blob/master/src/frames.ts#L53
now that you mention it, since we no longer use the raw name we get from the browser we could just use the enum :)